media/js/firefox/family/default-open-first-detail.es6.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import isSupported from './supports-intersection-observer.es6';
+import isAllowed from './allows-motion.es6';
+
+let _observer;
+let _faqSection;
+
+const createObserver = () => {
+    return new IntersectionObserver(
+        function (entries) {
+            entries.forEach(function (entry) {
+                if (entry.isIntersecting) {
+                    // open first details section
+                    const _firstDetail = _faqSection.querySelector('details');
+                    if (_firstDetail) {
+                        _firstDetail.setAttribute('open', '');
```

Comment:
It's not open by default on desktop for me.

Reply:
I'm having trouble reproducing this. What browser/screen size is not opening? Any chance you have a reduced motion pref on?

Reply:
It's only a problem for me locally, it works fine on the demo. I'm using Nightly at 1260px (but widening the screen and refreshing did not fix it).

Reply:
Curiouser and curiouser. It's a nice-to-have and not a requirement, but never fun to discover a "sometimes" bug. I may file a follow-up issue if this is still popping up once merged to main.